### DISCUSSION TITLE
Any way to determine compatibility?

### DISCUSSION THREAD

**Discussion by Kaan88** [roleoroleo/yi-hack-Allwinner-v2#252] [Q&A]
Hi,

I found Yi 1080 Home camera but the serials don't match with any of the projects.

Here is the picture, 
![image](https://user-images.githubusercontent.com/1260152/132412502-ae267b21-75af-4c91-9f23-82b76bfb5e47.png)

First 4 letters are 9y20, since I can't return it I want to make sure. Thanks.

**Comment by roleoroleo** [comment]
It's not possible to know the model without pairing with the app.
It seems a y20ga model but I can't be sure.

**Reply by RoosterX** [reply]
Since I have the same question, with same bundle deal/camera but different serial, I've tried to figure out where do I find the model number?
